openpype/settings/defaults/project_settings/maya.json

```
@@ -500,7 +500,24 @@
                     "shadows": true,
                     "textures": true,
                     "twoSidedLighting": true,
+                    "lineAAEnable": true,
                     "ssaoEnable": true,
+                    "ssaoAmount": 1,
+                    "ssaoRadius": 16,
+                    "ssaoFilterRadius": 16,
+                    "ssaoSamples": 16,
+                    "fogging": true,
+                    "hwFogFalloff": "0",
+                    "hwFogDensity": 0,
+                    "hwFogStart": 0,
+                    "hwFogEnd": 100,
+                    "hwFogAlpha": 0,
+                    "hwFogColorR": 1.0,
+                    "hwFogColorG": 1.0,
+                    "hwFogColorB": 1.0,
+                    "motionBlurEnable": true,
```

This should be off by default too

```suggestion
                    "motionBlurEnable": false,
```